provider_shopper/test/cart_widget_test.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+import 'package:provider_shopper/models/cart.dart';
+import 'package:provider_shopper/models/catalog.dart';
+import 'package:provider_shopper/screens/cart.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  List<SingleChildCloneableWidget> _providers = [
```

Comment:
Instead of `SingleChildCloneableWidget` use `SingleChildWidget`. The `SingleChildCloneableWidget` is removed in the recent version of `Provider`. You can refere the readme.

Reply:
Cool. Understandable changes. Will update.